PowerEditor/installer/nativeLang/german.xml

```diff
@@ -1104,20 +1105,28 @@ Translation note:
 					<Item id="4010" name="Registrierte Erw.:"/>
 				</FileAssoc>
 
+
 				<Language title="Sprache">
 					<Item id="6505" name="Verfügbar:"/>
 					<Item id="6506" name="Deaktiviert:"/>
 					<Item id="6507" name="Sprachmenü verschachtelt darstellen"/>
 					<Item id="6508" name="Sprachmenü"/>
+					<Item id="6335" name="Backslash ist Escapezeichen für SQL"/>
+				</Language>
+				
+				<Indentation title="Einrückungen">
+					<Item id="7161" name="Autom. Einrückung"/>
+					<Item id="7162" name="Keine"/>
+					<Item id="7163" name="Normal"/>
+					<Item id="7164" name="Erweitert"/>
 					<Item id="6301" name="Einstellung Einrückungen"/>
 					<Item id="6302" name="Leerzeichen"/>
 					<Item id="6303" name="Einrückungsbreite:"/>
 					<Item id="6310" name="Einrückung verwendet:"/>
 					<Item id="6311" name="Tab Zeichen"/>
 					<Item id="6510" name="Standard verwenden"/>
-					<Item id="6512" name="Die Rücktaste U+232B entfernt Einrückungen statt einzelne Leerzeichen"/>
-					<Item id="6335" name="Backslash ist Escapezeichen für SQL"/>
-				</Language>
+					<Item id="6512" name="Die Rücktaste U+232B entfernt Einrückungen statt einzelner Leerzeichen"/>
```

Comment:
It seems U+232B is not evaluated to the symbol (at least at my machine), but it works with this:
```
<Item id="6512" name="Die Rücktaste ⌫ entfernt Einrückungen statt einzelner Leerzeichen"/>
```
![image](https://github.com/user-attachments/assets/0ff3e76f-35da-4af4-9f17-67c7084b1e57)

Reply:
I would still vote just for:
"Rücktaste entfernt Einrückungen anstatt einzelner Leerzeichen"


Like commented here: https://github.com/notepad-plus-plus/notepad-plus-plus/pull/15353#discussion_r1652388836

The article takes unneccessary space and can be omitted. The symbol seems also not neccessary to me because "Rücktaste" is completely clear, but leave it if you think it's a must.

Reply:
Hi Markus, I also understand "Rücktaste" and it's completely obvious to me. However, other people I asked either called it "Backspace" (which I would not recommend in the German translation) or "Rückwärts löschen" (no comment on this one). "Rücktaste" however often left me looking in eyes questioning my sanity.
I'm not really into the Unicode character, but everyone seemed to understand this one, so I suggested it.